server/mappers/visitsWithVisitorsToListMapper.test.ts

```diff
@@ -11,7 +12,9 @@ describe('visitsWithVisitorsToListMapper', () => {
     expect(first.status).toEqual('Not entered')
     expect(first.isBooked).toEqual(false)
     expect(first.prison).toEqual('Moorland (HMP & YOI)')
-    expect(first.visitors).toEqual(['John Smith (Grandfather, 54 years old)'])
+    expect(first.visitors).toEqual([
+      `John Smith (Grandfather, ${ageAsString(pagedVisitsMock.content[0].visitors[0].dateOfBirth)})`,
+    ])
```

Comment:
I would suggest setting the system date e.g. `jest.useFakeTimers().setSystemTime(new Date('2020-01-10'))` rather than using code within tests